dev/com.ibm.ws.microprofile.config.1.3_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -123,8 +123,8 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.17</version>
                 <configuration>
+                    <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine> <!-- Needed for ZOS becuase arquillian calls string.getBtytes() -->
```

Comment:
This line is inside the <configuration> element but outside <systemPropertyVariables> whereas the one above is inside <systemPropertyVariables>

Reply:
This is the correct one. The one above has been fixed.